src/runtime/create-runtime-stories.ts

```diff
@@ -68,7 +68,10 @@ export const createRuntimeStories = (Stories: Component, meta: Meta<Cmp>) => {
        * The 'play' function should be delegated to the real play Story function
        * in order to be run into the component scope.
        */
-      storyObj.play = (storyContext) => {
+      function params(fn) { return fn.toString().match(/[^(]*\(([^)]*)/)?.slice(1) ?? [] }
+      const isMounting = params(storyObj.play).filter((p) => /\{\s*mount\s*\}/.test(p)).length != 0
+      storyObj.play = isMounting ? function ({ mount }) { return playDelegator(arguments[0]) } : (storyContext) => playDelegator(storyContext);
```

Comment:
I used arguments as destructuring `...var` rest and then spread many miss any non own enumerable props.